packages/flutter/lib/src/material/page.dart

```diff
@@ -17,14 +17,16 @@ class _MountainViewPageTransition extends AnimatedWidget {
     end: FractionalOffset.topLeft
   );
 
+  Animation<double> routeAnimation;
+
   _MountainViewPageTransition({
```

Comment:
constructor should be first in the class (so that it's obvious there's no implicit constructor)